arch/risc-v/src/c906/hardware/c906_clint.h

```diff
@@ -0,0 +1,36 @@
+/****************************************************************************
+ * arch/risc-v/src/c906/hardware/c906_clint.h
+ *
+ * Derives from software originally provided by Canaan Inc
+ *
+ *   Copyright 2018 Canaan Inc
```

Comment:
There is nothing in this file that needs to be attributed to Canaan

Reply:
Those files are derived/copied from K210, let me fix these. Thanks!